recipes/feather-format/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Darwin" ]; then
+    export MACOSX_DEPLOYMENT_TARGET=10.7
+    export CFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11"
```

Comment:
It's pretty weird putting this stuff in `CFLAGS`, but it seemed to ignore it otherwise. If someone has some better ideas here, that would be great. Maybe `feather-format` needs to be using `CXXFLAGS` or it calls it something else (`CPPFLAGS`?).